include/crypto.h

```diff
@@ -111,6 +111,7 @@ uint16_t Crypto_Calc_FECF(uint8_t* ingest, int len_ingest);
 void Crypto_Calc_CRC_Init_Table(void);
 uint16_t Crypto_Calc_CRC16(uint8_t* data, int size);
 int32_t Crypto_Check_Anti_Replay(SecurityAssociation_t *sa_ptr, uint8_t *arsn, uint8_t *iv);
+int32_t Crypto_Get_Acs_Algo(int8_t algo_enum);
```

Comment:
This function either needs to not depend on libgcrypt enums, or be moved into the cryptography interface where it has access to them.

Reply:
All noted, thanks for the feedback!